packages/date-picker/src/utils/formatDateAsText.ts

```diff
@@ -1,6 +1,6 @@
 import { format } from "date-fns"
```

Comment:
Can we replace this lib with [Intl.DateTimeFormat](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/DateTimeFormat)?